.storybook/webpack.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = ({ config, mode }) => ({
         },
       },
       {
-        test: /\.stories\.tsx?$/,
+        test: /(.?stories|story)\.tsx?$/,
```

Comment:
How about `/(\.?stories|story)\.tsx?$/` ?

Reply:
Do you want change this line?

Reply:
Yes, I thought the first dot should be escaped.

Reply:
Ok